cost/scheduled_reports/scheduled_report.pt

```diff
@@ -0,0 +1,306 @@
+name "Scheduled Report"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy allows you to set up scheduled reports that will provide summaries of cloud cost across all resources in the billing centers you specify, delivered to any email addresses you specify. The policy will report the following:
```

Comment:
Should the short description be this long and the long description be so short?

Reply:
changed